### ISSUE TITLE
 The (desired) drop-down zoom menu

### ISSUE THREAD

**Issue by leftCoast** [wokwi/wokwi-features#127]
There is a problem, at least on the Mac platform, where scrolling the objet pane in the X & Y axis causes uncontrollable jumps in the Z axis. In layman's terms, it zooms in and out uncontrollably and makes the UI ver hard to use. The proposed fix would be to add something like a zoom-level popup menu to this edit screen and remove the zooming ability from the dragging action.

**Comment by urish** [comment]
This has been bumped to the top of the [feature list](https://wokwi.com/features), starting to work on it

**Comment by leftCoast** [comment]
Oh no you don't! Gotta' finish what you started. Here's $4 more to keep
you on target. :)

-jim lee

**Comment by urish** [comment]
We now have this menu!  🎉

Also, panning and zooming with the mouse should have been improved.

**Comment by leftCoast** [comment]
got it
